benchmarks-intellij/src/test/kotlin/com/jetbrains/compose/color/ColorPickerUITest.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020-2022 JetBrains s.r.o. and respective authors and developers.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package com.jetbrains.compose.color
```

Comment:
Can we delete this file and HSVTest.kt? I suppose this is from another project.

Reply:
Yes, sure!

Reply:
Done